### PR TITLE
[WIP] openjdk14

### PR DESCRIPTION
**Pull request by sgn**
- Should be merged in 2 steps, first step with 4 first commits, the second step to enable cross

### CODE REVIEW

File: srcpkgs/openjdk14/template [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+case "$XBPS_TARGET_MACHINE" in
+	ppc64*) ;;
+	ppc*) _use_zero=yes ;;
+esac
+
+if [ -n "$_use_zero" ]; then
+	makedepends+=" libffi-devel"
+	configure_args+=" --with-jvm-variants=zero"
+	case "$XBPS_TARGET_MACHINE" in
+	ppc*) configure_args+=" --with-boot-jdk-jvmargs=-XX:ThreadStackSize=2560";;
```
Comment (fosslinux) [inline srcpkgs/openjdk14/template:60]: Indent?

File: srcpkgs/openjdk14/template [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+	esac
+fi
+
+case "$XBPS_TARGET_LIBC" in
+	glibc) build_options_default+=" docs";;
```
Comment (fosslinux) [inline srcpkgs/openjdk14/template:65]: Why is this?

File: srcpkgs/openjdk14/template [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+ jdk:/usr/bin/rmic:/usr/lib/jvm/openjdk14/bin/rmic
+ jdk:/usr/bin/rmid:/usr/lib/jvm/openjdk14/bin/rmid
+ jdk:/usr/bin/rmiregistry:/usr/lib/jvm/openjdk14/bin/rmiregistry
+ jdk:/usr/bin/serialver:/usr/lib/jvm/openjdk14/bin/serialver
+ jdk:/usr/bin/unpack200:/usr/lib/jvm/openjdk14/bin/unpack200
+"
+
+post_extract() {
+	chmod +x configure
+	if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
```
Comment (fosslinux) [inline srcpkgs/openjdk14/template:121]: Ideally, keep this consistent with prior XBPS_TARGET_LIBC checks, use `case` or `if` the whole way though.

**Comment by fosslinux** [comment]
??

**Comment by sgn** [comment]
Due to PEBCAK, I executed some cleanup tools :-p

**Pull request closed without merging**